integr8ly/tutorial-web-app
# Branding update

## Motivation
https://issues.redhat.com/browse/IUXD-74?jql=labels%20%3D%20integration-ux-dev%20AND%20assignee%20in%20(christiemolloy)

## What
function added to `masthead.js` and `aboutModal.js` to return different logos depending on the cluster type.

## Why
Logos need to be changed depending on the cluster type.

## How
function added to `masthead.js` and `aboutModal.js` to return different logos depending on the  cluster type.

## Verification Steps
1. Go to `masthead.js`
2. Go to `getLogo`
3. Add a new line to make the cluster type: `poc` `osd` `pds` `localhost`
4. Check that the corresponding logos match the cluster type

## Checklist:


## Progress


## Additional Notes

<img width="1276" alt="Screen Shot 2019-12-19 at 10 44 47 AM" src="https://user-images.githubusercontent.com/20118816/71188385-df74f900-224e-11ea-9025-e61c8b1f15a5.png">
<img width="1241" alt="Screen Shot 2019-12-19 at 10 45 19 AM" src="https://user-images.githubusercontent.com/20118816/71188386-df74f900-224e-11ea-9091-5773a6a6486d.png">
<img width="934" alt="Screen Shot 2019-12-20 at 10 55 28 AM" src="https://user-images.githubusercontent.com/20118816/71266867-49f16c00-2317-11ea-8560-3478ef624701.png">
<img width="914" alt="Screen Shot 2019-12-20 at 10 53 53 AM" src="https://user-images.githubusercontent.com/20118816/71266877-4f4eb680-2317-11ea-806f-705f40e9bbe5.png">
